src/lib/pages/frontsite/SelfCheckout/state/actions.js

```diff
@@ -51,20 +43,14 @@ export const selfCheckOutSearch = (term) => {
   };
 };
 
-export const checkoutItem = (documentPid, itemPid, patronPid) => {
+export const selfCheckOut = (documentPid, itemPid, patronPid) => {
```

Comment:
The related tests in `actions.test.js` are not updated (we don't use the item anymore),  and they are failing.